test/integration/stats_integration_test.cc

```diff
@@ -25,6 +32,34 @@ class StatsIntegrationTest : public testing::TestWithParam<Network::Address::IpV
   void initialize() override { BaseIntegrationTest::initialize(); }
 };
 
+class ClusterMemoryUtilization : public testing::TestWithParam<Network::Address::IpVersion>,
```

Comment:
I meant to also comment on this: this should not inherit from BaseIntegrationTest; it should just inherit from TestWithParam.

Reply:
I tried to tried to not derive from BaseIntegrationTest but the config_helper_ within is protected and I couldn't figure out a way around this other than possibly listing it as a friend

Reply:
I think you could still derive a class from BaseIntegrationTest which you instantiate, and you could access config_helper_ from that class.

Reply:
Would this be like 

```
auto t1 = std::make_unique<BaseIntegrationTest>(IpVersions);
t1 -> config_helper_.addConfigModifier(...) ;
```